libkbfs/journal_block_server_test.go

```diff
@@ -35,7 +35,8 @@ func setupJournalBlockServerTest(t *testing.T) (
 	// Clean up the config if the rest of the setup fails.
 	defer func() {
 		if !setupSucceeded {
-			CheckConfigAndShutdown(t, config)
+			ctx := context.Background()
```

Comment:
Why not return a `ctx` from `setupJournalBlockServerTest` and use it everywhere, like in the other test files? And maybe put a test timeout on it while you're at it?

(Same for the other journal test files below.)